test-suite/testheaders.sh

```diff
@@ -36,16 +33,15 @@ for f in $@; do
 int main( int argc, char* argv[] ) { return 0; }
 EOF
         if ${cc} -c -o $t.o $t.cc ; then
-            echo "Ok."
+            echo "ok ${testNum} - ${f}"
         else
-            echo "Fail."
+            echo "not ok ${testNum} - ${f}"
             exitCode=1
         fi
         rm $t.cc $t.o
+        testNum=`expr ${testNum} + 1`
     fi
-    test $exitCode -eq 0 || break
 done
 
 #who ever said that the test program needs to be meaningful?
```

Comment:
This comment is made stale by the changed line below.
```suggestion
```

PS. removing this `cp` resolves https://bugs.squid-cache.org/show_bug.cgi?id=5237